compose-dev-frontend.yml

```diff
@@ -13,8 +13,8 @@ services:
       dockerfile: Dockerfile
       args:
         - SKIP_BACKEND_TEST=true
-        - SKIP_FRONTEND_BUILD=true
-        # - NODE_ENV=development
+        # - SKIP_FRONTEND_BUILD=true
```

Comment:
I think skipping frontend build should be enabled by default here, it's done that way to reduce the build time significantly.

Reply:
dropped this change